src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,78 @@
+import type { FC, ReactNode } from 'react';
```

Comment:
I think I'm leaning towards putting all components in a title case file (i.e. `Tooltip.tsx`) and just using `index.ts` for re-exporting. Makes the DX a little bit nicer.